tutorials/kotlin-springboot-app-engine-java8.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Run a Kotlin Spring Boot app on Google App Engine Standard
+description: Learn how to deploy a Kotlin Spring Boot app to Google App Engine Standard.
+author: bshaffer
+tags: App Engine, Kotlin, Spring Boot, Java, App Engine Standard
+date_published: 2018-01-17
+---
+
+[Google App Engine Standard](https://cloud.google.com/appengine/docs/standard/)
+is an easy way to deploy your apps to the same infrastructure that powers
```

Comment:
We try to avoid claims of difficulty.

Reply:
This change is inadvertent.

Reply:
This was not my change not sure how it got into the PR.  may need to close and reopen.